### PR TITLE
[VSCode Extension] Add missing files & Configure CI

### PR DESCRIPTION
**Pull request by yatarkan**

### CODE REVIEW

**Review by apaniukov — APPROVED** [review]

File: .github/workflows/openvino_code.yml [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+jobs:
+  check_extension:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ github.event.pull_request.head.ref }}
+          repository: ${{github.event.pull_request.head.repo.full_name}}
```
Comment (ilya-lavrenov) [inline .github/workflows/openvino_code.yml:26]: I suppose these explicit `ref` and `repository` are not required

**Review by apaniukov — APPROVED** [review]

File: .github/workflows/openvino_code.yml [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+on:
+  pull_request:
+    types: [opened, synchronize]
+    paths:
+      - 'modules/openvino_code/**'
+      - '.github/workflows/openvino_code.yml'
+
+concurrency:
+  group: ${{ github.head_ref || github.ref_name }}
```
Comment (akashchi) [inline .github/workflows/openvino_code.yml:11]:
```suggestion
  group: ${{ github.head_ref || github.ref_name }}-openvino-code
```

That would future-proof this pipeline if other pipelines are implemented with `concurrency`. `github.head_ref` and `github.ref_name` are not unique for the workflow.

File: .github/workflows/openvino_code.yml [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+defaults:
+  run:
+    working-directory: ./modules/openvino_code
+
+jobs:
+  check_extension:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
```
Comment (akashchi) [inline .github/workflows/openvino_code.yml:23]:
```suggestion
        uses: actions/checkout@v4
```

File: .github/workflows/openvino_code.yml [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+    
+      - name: Set up Python 3.8
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.8'
+          cache: 'pip'
+
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
```
Comment (akashchi) [inline .github/workflows/openvino_code.yml:58]: No need as `actions/setup-python@v4` is updating `pip`.

File: .github/workflows/openvino_code.yml [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+        run: npm ci
+
+      - name: Run Lint
+        run: npm run lint:all
+  
+  check_server:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
```
Comment (akashchi) [inline .github/workflows/openvino_code.yml:45]:
```suggestion
        uses: actions/checkout@v4
```

File: .github/workflows/openvino_code.yml (hunk 2 above)
Comment (yatarkan) [reply .github/workflows/openvino_code.yml:11]: Fixed

File: .github/workflows/openvino_code.yml (hunk 5 above)
Comment (yatarkan) [reply .github/workflows/openvino_code.yml:45]: Fixed

File: .github/workflows/openvino_code.yml (hunk 1 above)
Comment (yatarkan) [reply .github/workflows/openvino_code.yml:26]: Fixed

**Review by ilya-lavrenov — APPROVED** [review]

**Review by akashchi — APPROVED** [review]

**Review by as-suvorov — APPROVED** [review]

**Pull request merged**